lib/ProductOpener/APIProductWrite.pm

```diff
@@ -217,9 +217,7 @@ sub update_tags_fields ($request_ref, $product_ref, $field, $add_to_existing_tag
 		my $tags_list = join(',', @$value);
 
 		if ($remove_tags && ref($remove_tags) eq 'ARRAY') {
-			foreach my $tag (@$remove_tags) {
-				remove_tags_from_field($product_ref, $tags_lc, $field, $tag);
-			}
+			remove_tags_from_field($product_ref, $tags_lc, $field, $remove_tags);    #join(',', @$remove_tags)
```

Comment:
you have a ref to an array but then in remove_tags_from_field you treat it like a string… it wont work ! Why don't you use a join ?

Reply:
Because I'm calling the join function inside the remove_tags_from_field function